test/cppsuite/test_harness/workload/workload_validation.h

```diff
@@ -41,7 +41,9 @@ extern "C" {
 
 namespace test_harness {
 struct key_state {
-    bool exists;
+    key_state() = default;
```

Comment:
Why do we declare the default constructor here and not for `collection_cursor` ?

Reply:
It's because we're storing it in a `std::map`. If we want to use the `[]` operator with the map, the value needs to have a default constructor.